lms/services/document_url.py

```diff
@@ -93,17 +93,18 @@ def _from_assignment_in_db(self, request):
         for param in (
             "resource_link_id",  # A normal LTI (non-deep linked) launch
             "resource_link_id_history",  # A Blackboard course we can copy
-            "ext_d2l_resource_link_id_history",  # Ditto for Brightspace
+            "ext_d2l_resource_link_id_history",  # Ditto for D2L
```

Comment:
D2L it's just sorter for the next line

Reply:
Is it worth expanding the `"resource_link_id_history"` and `"ext_d2l_resource_link_id_history"` params to say that these are used by Blackboard and D2L _in LTI 1.1_?